Src/WitsmlExplorer.Api/Models/Reports/DeleteEmptyMnemonicsReport.cs

```diff
@@ -0,0 +1,14 @@
+using WitsmlExplorer.Api.Jobs.Common;
+
+namespace WitsmlExplorer.Api.Models.Reports
+{
+    public class DeleteEmptyMnemonicsReport : BaseReport
+    {
+    }
+
+    public class DeleteEmptyMnemonicsReportItem
```

Comment:
DeleteEmptyMnemonicsReportItem should consist of string fields, because it corresponds to a single row of the resulting csv report. The fields should be: well name, well uid, wellbore name, wellbore uid, log name, log uid, log index type, mnemonic.